R/tokenization.R

```diff
@@ -152,3 +155,60 @@ wordpiece_tokenize <- function(text,
   }
   return(sub_tokens)
 }
+
+
+# .process_vocab -----------------------------------------------------------
+
+.process_vocab <- function(v) {
+  UseMethod(".process_vocab", v)
+}
+
+.process_vocab.wordpiece_vocabulary <- function(v) {
+  .process_wp_vocab(v)
+}
+
+.process_vocab.character <- function(v) {
+  v
+}
+
+.process_wp_vocab <- function(v) {
+  UseMethod(".process_wp_vocab", v)
+}
+
+.process_wp_vocab.wordpiece_vocabulary <- function(v) {
```

Comment:
I'm pretty sure this is extraneous; if we don't have this method, it'll automatically go to the next one. Test to make sure. I think by having it we might technically (barely) slow the process down.